.github/workflows/pull-request-checks.yaml

```diff
@@ -365,7 +365,9 @@ jobs:
         with:
           submodules: recursive
       - name: Download test dependencies
-        run: sudo apt install openjdk-11-jdk-headless
+        run: |
+          sudo apt update
+          sudo apt install openjdk-11-jdk-headless
```

Comment:
Should our other jobs get the same treatment?

Reply:
Probably.

Reply:
@hannes-steffenhagen-diffblue Okay I applied it to all the jobs.